### PR TITLE
Cowboy hats resprite

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
resprites cowboy hats with a bonus - security cowboy hat (because when you are restricted on what you can wear, you need as many options as possible)

## Why It's Good For The Game
yeeeehaaaw

## Images of changes
image

## Testing
tested on local

## Changelog
:cl:
add: New headwear for security - security cowboy hat
imageadd: New sprites for all cowboy hats
/:cl:

### CODE REVIEW

**Comment by TravisAngeI** [comment]
those item sprites look very nice

File: code/modules/client/preference/loadout/loadout_hat.dm [hunk 1]
```diff
@@ -72,6 +72,12 @@
 	path = /obj/item/clothing/head/soft/sec/corp
 	allowed_roles = list("Head of Security", "Warden", "Security Officer")
 
+/datum/gear/hat/cowboyhat/sec
+	display_name = "Cowboy hat, security"
+	path = /obj/item/clothing/head/cowboyhat/sec
+	allowed_roles = list("Head of Security", "Warden", "Security Officer")
+
+
```
Comment (1080pCat) [inline code/modules/client/preference/loadout/loadout_hat.dm:80]:
```suggestion

```

**Comment by Henri215** [comment]
image

Is it missing a pixel?

**Review by DGamerL — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]
https://www.youtube.com/watch?v=uc1FuP9LuPY

**Pull request merged**